ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRaoResult.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *  * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  * This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.farao_community.farao.linear_rao;
+
+import com.farao_community.farao.rao_api.RaoResult;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.powsybl.commons.extensions.AbstractExtension;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+public class LinearRaoResult extends AbstractExtension<RaoResult> {
+
+    public enum SystematicSensitivityAnalysisParameters {
+        DEFAULT,
+        FALLBACK
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private SystematicSensitivityAnalysisParameters systematicSensitivityAnalysisParameters;
+
+    @Override
+    public String getName() {
+        return "LinearRaoResult";
+    }
+
+    public SystematicSensitivityAnalysisParameters getSystematicSensitivityAnalysisParameters() {
+        return systematicSensitivityAnalysisParameters;
+    }
```

Comment:
It's just about the wording, but I would have prefer SystematicSensitivityAnalysis**Status**.
Parameters make me think about input configuration parameters.

Reply:
and maybe with three values ? DEFAULT, FALLBACK or FAILURE ? depending if the sensi fails (whatever the config), or if it works with default or fallback parameters.